src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs

```diff
@@ -23,6 +23,7 @@ public class MvcRazorTemplateEngine : RazorTemplateEngine
             RazorProject project)
             : base(engine, project)
         {
+            Options.ImportsFileName = "_ViewImports.cshtml";
```

Comment:
This should not break anything. The Page loader sets this value explicitly. In fact both places in MVC where this class are used set it.

I'm working on some other changes to MVC to streamline this.